go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/stretchr/testify v1.4.0 // indirect
 )
+
+go 1.13
```

Comment:
can you expand on why this is required? will we have to keep changing this when we advance to 1.14 and beyond?